x/examples/outline-connectivity/main.go

```diff
@@ -206,34 +182,31 @@ func main() {
 				DurationMs: testDuration.Milliseconds(),
 				Error:      makeErrorRecord(testErr),
 			}
-			err := jsonEncoder.Encode(record)
+			collectorURL, err := url.Parse(*reportToFlag)
 			if err != nil {
-				log.Fatalf("Failed to output JSON: %v", err)
+				debugLog.Printf("Expected no error, but got: %v", err)
```

Comment:
This is more informative:
```suggestion
				debugLog.Printf("Failed to parse collector URL: %v", err)
```